plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/metastore/HiveMetastoreBackedDeltaLakeMetastore.java

```diff
@@ -182,6 +182,15 @@ public void dropTable(ConnectorSession session, String databaseName, String tabl
         }
     }
 
+    @Override
+    public void unregisterTable(ConnectorSession session, String databaseName, String tableName)
```

Comment:
nit: the content of this method is almost 1-to-1 with `io.trino.plugin.deltalake.metastore.HiveMetastoreBackedDeltaLakeMetastore#dropTable` .
Can we extract this logic to a common private method?